cmd/stackdriver-prometheus-sidecar/main.go

```diff
@@ -293,6 +300,20 @@ func main() {
 	}
 
 	cfg.projectIdResource = fmt.Sprintf("projects/%v", *projectId)
+	if cfg.apiOverride {
+		// manual.GenerateAndRegisterManualResolver generates a Resolver and a random scheme.
+		// It also registers the resolver. rb.InitialAddrs adds the addresses we are using
+		// to resolve GCP API calls to the resolver.
+		cfg.manualResolver, _ = manual.GenerateAndRegisterManualResolver()
+		cfg.manualResolver.InitialAddrs([]resolver.Address{
```

Comment:
Please document these addresses. You can use the same comment from your other CL.